src/main/java/org/mifos/processor/bulk/camel/config/HttpClientConfigurerTrustAllCACerts.java

```diff
@@ -0,0 +1,73 @@
+package org.mifos.processor.bulk.camel.config;
+
+import java.security.KeyManagementException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLContext;
+import org.apache.camel.component.http.HttpClientConfigurer;
+import org.apache.http.config.Registry;
+import org.apache.http.config.RegistryBuilder;
+import org.apache.http.conn.socket.ConnectionSocketFactory;
+import org.apache.http.conn.socket.PlainConnectionSocketFactory;
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.apache.http.ssl.SSLContextBuilder;
+import org.apache.http.conn.ssl.TrustStrategy;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HttpClientConfigurerTrustAllCACerts implements HttpClientConfigurer {
+
+    public Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    public HttpClientConfigurerTrustAllCACerts() {
+    }
+
+    @Override
+    public void configureHttpClient(HttpClientBuilder clientBuilder) {
+        // setup a Trust Strategy that allows all certificates.
+        //
+        SSLContext sslContext = null;
+        try {
+            sslContext = new SSLContextBuilder().loadTrustMaterial(null, new TrustStrategy() {
+                public boolean isTrusted(X509Certificate[] arg0, String arg1) throws CertificateException {
+                    return true;
+                }
+            }).build();
+        } catch (KeyManagementException e) {
```

Comment:
combine all catches to one, use logger instead of stacktrace

Reply:
done